### PR TITLE
Implement Bento services for sklearn models

### PR DESCRIPTION
**Pull request by tacvoong**

### CODE REVIEW

**Review by htkhiem — CHANGES_REQUESTED** [review]
Please remove `_svc.py` from your commit. They're meant to be generated just for packaging into Bentos and then deleted from the outer filesystem. It's kind of a workaround for Bento service files not being flexible enough for our use.

File: onnx/bentoml/tfidf-hsgd/tfidf_hsgd.template.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+from nltk.tokenize import word_tokenize
+from sklearn import preprocessing, metrics
+from sklearn.pipeline import make_pipeline, Pipeline
+
+
+# Use load_runner for optimised performance
+# The transformer encoder section uses a directly-exported model, instead of
+# through ONNX, for optimisation.
+
+# PyTorch-based
```
Comment (htkhiem) [inline onnx/bentoml/tfidf-hsgd/tfidf_hsgd.template.py:21]: Do clean up unrelated comments and excess blank lines.

**Comment by tacvoong** [comment]
weird, i did not see them and i thought gitignore caught it

**Review by htkhiem — CHANGES_REQUESTED** [review]

**Review by htkhiem — COMMENTED** [review]
LGTM, will test run shortly. Don't merge until them.

**Review by htkhiem — APPROVED** [review]

**Pull request merged**